### DISCUSSION TITLE
coc#refresh() not triggering

### DISCUSSION THREAD

**Discussion by jmreicha** [neoclide/coc.nvim#3396] [Q&A] [answered]
Hoping this is an easy one, I have tested and the following snippet from the README works on OSX but for some reason (all other config is identical in .vimrc) the popup never triggers on my Linux machine.

```vim
" Use <c-space> to trigger completion.
if has('nvim')
  inoremap <silent><expr> <c-space> coc#refresh()
else
  inoremap <silent><expr> <c-@> coc#refresh()
endif
```

I should also mention that I am experiencing the issue on Windows Terminal, but I don't experience the issue with any of the other completion keys/commands, and I can also see the mapping if I do `:verbose imap <c-@>`. I've tried a handful of other mappings as well so is the `coc#refresh()` somehow not being triggered? Testing this with the `coc-go` extension.

Any ideas what is going wrong?

**Comment by chemzqm** [comment, marked as answer]
Check verbose log of coc.nvim https://github.com/neoclide/coc.nvim/wiki/F.A.Q#how-to-get-log-of-cocnvim
